src/table/table-model.class.ts

```diff
@@ -546,6 +546,22 @@ export class TableModel implements PaginationModel {
 		this.header[index].sorted = true;
 	}
 
+	/**
+   * reset the original sort
+   */
+	resetSort() {
+		this.pushRowStateToModelData();
+		this.data.sort((a, b) => a[0].originIdx - b[0].originIdx);
+		this.popRowStateFromModelData();
+		this.header.forEach(column => (column.sorted = false));
+
+		this.data.forEach(item => {
+			item.forEach(subItem => {
+				delete subItem.originIdx;
+			});
+		});
```

Comment:
Curious, do we _NEED_ to delete the index attribute from the cell object?

Reply:
yes, we need. otherwise, the test will failed.